_providers/hotmail.md

```diff
@@ -6,13 +6,29 @@ domains:
   - outlook.com
   - office365.com
   - outlook.com.tr
+  - live.com
+server:
+  - type: imap
+    socket: SSL
+    hostname: imap-mail.outlook.com
+    port: 993
+  - type: smtp
+    socket: STARTTLS
+    hostname: smtp-mail.outlook.com
+    port: 587
 before_login_hint: |
   Outlook email addresses will not work as expected as these servers remove some important transport information.
   Hopefully sooner or later there will be a fix, for now we suggest to use another email address.
+after_login_hint: |
+  Outlook email addresses will not work as expected as these servers remove some important transport information.
+  Unencrypted 1-on-1 chats kind of work, but groups and encryption don't.
+  Hopefully sooner or later there will be a fix, for now we suggest to use another email address.
+  You can compare email providers at https://providers.delta.chat :)
```

Comment:
I'd like to leave this line out for now. This data is going live soon, but providers.delta.chat currently doesn't show the state of any provider.

Additionally, providers.delta.chat isn't very helpful in actually comparing providers. It only shows an overview which providers work with Delta Chat — which is a fine and well-restricted purpose for a simple website, but not a comparison aid. So we should think with a different wording anyway, something like
> You may check in advance if your newly chosen provider works well with Delta Chat at https://providers.delta.chat/

Reply:
Sounds good. I'll fix the status issue, then it doesn't speak anything against guiding people to the overview site, right? Better than not offering it, I'd say.

Reply:
The overview page is out of scope currently, let's speak about that when the current project is finished.